local-tests/test_runtime_update.py

```diff
@@ -0,0 +1,113 @@
+#!/bin/env python
```

Comment:
I'll review this file after first review round :)

Reply:
now you have much less to review :)